security/advancedtls/crl.go

```diff
@@ -55,8 +55,9 @@ type Cache interface {
 	Get(key any) (value any, ok bool)
 }
 
-// RevocationConfig contains options for CRL lookup.
-type RevocationConfig struct {
+// RevocationOptions allows a user to configure revocation using certificate
```

Comment:
I'd word it like 'RevocationOptions allows users to configure certification revocation behavior' - WDYT?

Reply:
I like that more, changed